locale/de_DE/grid.xml

```diff
@@ -225,10 +225,10 @@
 	<message key="grid.user.addRole">Rolle hinzufügen</message>
 	<message key="grid.user.selectUser">Benutzer/in auswählen</message>
 	<message key="grid.user.pleaseSelectUser">Bitte wählen Sie eine/n Benutzer/in aus.</message>
-	<message key="grid.user.confirmLogInAs">Log in as this user? All actions you perform will be attributed to this user.</message>
+	<message key="grid.user.confirmLogInAs">Einloggen als diesen Benutzer/dieser Benutzerin? Alle Aktionen, die Sie durchführen, werden diesem Benutzer/dieser Benutzerin zugeordnet.</message>
 	<message key="grid.user.mergeUsers">Benutzer/innen zusammenführen</message>
 	<message key="grid.user.mergeUsers.mergeUser">Benutzer/in zusammenführen</message>
-	<message key="grid.user.mergeUsers.mergeIntoUser">Merge into this User</message>
+	<message key="grid.user.mergeUsers.mergeIntoUser">Zu diesem Benutzer/dieser Benutzerin zusammenführen</message>
```

Comment:
Vielleicht etwas expliziter: "In diesem Benutzerkonto aufgehen lassen"? (see `grid.user.mergeUsers.confirm`)? So ist es grammatikalisch und semantisch etwas schwerfällig…

Reply:
Ich habe jetzt einmal "In diesem Benutzerkonto zusammenführen" gewählt.